<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_Sex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пол:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Sex_m">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>М</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Sex_W">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Str">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сила</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ловкость</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Intel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Интеллект</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Str">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Cha">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очарование</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Points">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Points">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>20</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Здоровье (HP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP_Value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MP">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Мана (MP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Def">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Защита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Atra">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Привлекательность</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Att">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Атака</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Def_Value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MP_Value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Att_Value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Atra_Value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Class">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Класс:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Class_Value">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Неудачник</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Dex">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Cha">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Intel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Luck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Удача</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Luck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ERROR">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ERROR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
